x-pack/legacy/plugins/reporting/server/legacy.ts

```diff
@@ -41,11 +43,17 @@ export const legacyInit = async (
     server.newPlatform.coreContext as PluginInitializerContext,
     buildConfig(coreSetup, server, reportingConfig)
   );
+
   await pluginInstance.setup(coreSetup, {
     elasticsearch: coreSetup.elasticsearch,
     security: server.newPlatform.setup.plugins.security as SecurityPluginSetup,
     usageCollection: server.newPlatform.setup.plugins.usageCollection,
     __LEGACY,
+    licensing: {
```

Comment:
Need to verify this is working properly as I'm not fully versed on how this works